src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,1017 @@
+
+Probes are electrode-bearing devices that interface with neural tissue to record electrophysiological activity, ranging from multi-electrode arrays to single recording pipettes. They can be permanently implanted (chronic recordings) or inserted temporarily for the recording (acute recordings).
+
+The probe positions and properties are stored in a `.tsv` file.
+This file contains the probe ID, the type of recording (acute/chronic), and the probe coordinates.
+
+### ProbeInterface Library
+
+[ProbeInterface](https://github.com/SpikeInterface/probeinterface) is a standard for specifying electrode layouts on probes.
+The [ProbeInterface library](https://github.com/SpikeInterface/probeinterface_library) includes layouts for many common probes.
+
+Probe information is specified in the `probes.json` sidecar file using the `model` field with `Levels` to define each probe model.
+
+For probes listed in the ProbeInterface library, use `TermURL` to reference the probe definition:
+
+```json
+"model": {
+    "Levels": {
+        "A1x32": {
+            "Description": "A1x32-Poly3-10mm-50-177, a 1-shank probe",
+            "TermURL": "https://raw.githubusercontent.com/SpikeInterface/probeinterface_library/refs/heads/main/neuronexus/A1x32-Poly3-10mm-50-177/A1x32-Poly3-10mm-50-177.json"
+        }
+    }
+}
+```
+
+If the probe is not listed in the ProbeInterface library, you SHOULD define it using the [ProbeInterface format](https://probeinterface.readthedocs.io/en/latest/format_spec.html) and include it in a directory called `probes/` in the root of the dataset. Custom probe files MUST comply with the [ProbeInterface specification](https://probeinterface.readthedocs.io/en/latest/format_spec.html) and [JSON schema](https://raw.githubusercontent.com/SpikeInterface/probeinterface/refs/heads/main/src/probeinterface/schema/probe.json.schema).
+
+For custom probes, reference them using a [BIDS URI](../common-principles.md#bids-uri) with the `bids::` prefix in the `TermURL` field:
+
+```json
+"model": {
+    "Levels": {
+        "customprobe1": {
+            "Description": "Custom experimental probe",
+            "TermURL": "bids::probes/customprobe1.json"
+        }
+    }
+}
+```
+
+Example file structure:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example(
+   {
+   "probes": {
+      "customprobe1.json": "",
+      "customprobe2.json": "",
+      "...": "",
+      },
+   }
+) }}
+
+{{ MACROS___make_columns_table("microephys.microephysProbes") }}
+
+### Example `*_probes.tsv`
+
+**Extracellular electrophysiology example:**
+
+```tsv
+probe_name	type	AP	ML	DV	AP_angle	ML_angle	rotation_angle	hemisphere	manufacturer	device_serial_number	electrode_count	width	height	depth	dimension_unit	coordinate_reference_point	associated_brain_region	associated_brain_region_id	reference_atlas	material
```

Comment:
note: at our meeting Oct 15th 2025 we briefly touched on a possible topic of introducing `probe-<label>` entity! It would reflect that actual recordings directly from hardware do ATM store each probe separately and it could be that there are different manufacturers/types of probes.  That would then mandate pretty much to have `probe_id` (`probe-<label>`) as the leading column in the `*probes.tsv` file (which would exist then to summarize across probes, potentially even may be benefitting from inheritance principle for common metadata?).

Here `probe_name` came from analogy to `name` of `electrodes.tsv` inherited from EEG.  But the question remains whether we might want to standardize this right away.  WDYT @effigies ?  Should we may be right away introduce `probe` entity?